stylesheets/shared/layout.css

```diff
@@ -30,6 +30,9 @@
     padding-top: 1rem;
     padding-bottom: 1rem;
 }
+.pd-x-t-1{
```

Comment:
Better to use as less as possible. For eg, it can be .pd-t-1 ig

Reply:
Follow the x & y direction naming scheme